src/modinv32_impl.h

```diff
@@ -584,4 +661,69 @@ static void secp256k1_modinv32_var(secp256k1_modinv32_signed30 *x, const secp256
     *x = d;
 }
 
+/* Compute the Jacobi symbol of x modulo modinfo->modulus (variable time). gcd(x,modulus) must be 1. */
```

Comment:
Nits (also applies to `modinv64_impl`):
- Is `gcd(x,modulus) must be 1` still true, since you're handling `x=0` here?
- Is it necessary to also mention `x must be normalized` here for consistency with `modinv32.h`?

Reply:
I made a few changes to address this.